codesign/lib/src/utils.dart

```diff
@@ -29,6 +29,21 @@ enum FileType {
   }
 }
 
+Future<bool> isSymlink(String fileOrFolderPath, ProcessManager processManager) async {
```

Comment:
You should avoid calling out to sub-processes and parsing their output when possible. You can check if it's a link from a package:file entity by checking if it's a `Link` https://pub.dev/documentation/file/latest/file/Link-class.html

Reply:
specifically this method: https://api.dart.dev/stable/2.17.5/dart-io/FileSystemEntity/isLinkSync.html

Reply:
Yeah I tried to use Link previously, but the fileSystemEntity does not return true in the check of `entity is io.Link` or `entity is Link`, when the FileSystemEntity is a link. e.g. listing a folder A which contains child file symlink B, and B would fail the check of `B is Link`. umm maybe there is a better way to check if an entity resolves to Link rather than `is`?

Reply:
check my previous comment

Reply:
Ahhh there is a new comment every time I refresh. Thanks a lot Christopher!

Reply:
Done.